### ISSUE TITLE
能否支持early stop？

### ISSUE THREAD

**Issue by ownnaruto** [GestaltCogTeam/BasicTS#128]
您好，感谢您提供这么完整，清晰的框架。
我注意到目前框架在训练时是固定训练100个epoch，这样是否可能会存在训练不充分的问题？（比如STID 在METR-LA数据集上100个epoch似乎并未训练充分，可能效果还能再继续提升一些），能否加入early stop机制呢？

**Comment by zezhishao** [comment]
感谢你的建议，这一功能确实很实用。这个功能更新起来有点复杂，我们会在下一个大版本更新中加入这个功能。
目前想要缓解的话，可以考虑将`CFG.TRAIN.NUM_EPOCHS`设置得大一点。假如是训练完之后才发现不够的话，可以在配置文件中设置`CFG.TRAIN.FINETUNE_FROM`，让模型继续训练（注意可能需要同步调整 learning rate）。

**Comment by zezhishao** [comment]
您好，目前已经支持EarlyStop机制。例如，`CFG.TRAIN.EARLY_STOPPING_PATIENCE = 10`。
